README.md

```diff
@@ -55,33 +55,32 @@ $ mate \
     --google-record-group-id=foo
 ```
 
-Analogous to the AWS case
+Analogous to the AWS case with the difference that it doesn't use the AWS specific Alias functionality but plain A records.
 
 ### Kubernetes
 
-By default Mate will retrieve the list of services from Kubernetes API server via `http://127.0.0.1:8001` (for local testing use `kubectl proxy`), however API server url can be configured with `kubernetes-server` flag. 
-Mate will listen for events from Kubernetes API Server and create corresponding records for newly created services. Further synchronization (create, removal, update) will occur every minute. If you only like to do the synchronization you can enable `sync-only` flag. 
+By default Mate will retrieve the list of services from the Kubernetes API server via `http://127.0.0.1:8001` (for local testing use `kubectl proxy`), however the API server url can be configured with the `kubernetes-server` flag.
+Mate will listen for events from the API Server and creates corresponding records for newly created services. Further synchronization (update and removal) will occur every minute. There's an initial syncronization when Mate boots up so it's safe to reboot the process at any point in time. If you only like to do the synchronization you can use the `sync-only` flag.
```

Comment:
` (update and removal) will occur every minute...` - i think create happens as well, just via upsert. This is especially relevant if `sync-only` is enabled.